gulp_tasks/postcss.js

```diff
@@ -10,6 +10,7 @@ const pkg = require( '../package.json' );
 const compilePlugins = [
 	require( 'postcss-partial-import' )( {
 		extension: '.pcss',
+		prefix: '_',
```

Comment:
the latest version of postcss-partial-import requires this value to find imports properly.